tests/ludwig/schema/test_model_metadata.py

```diff
@@ -0,0 +1,133 @@
+from typing import Union
+from ludwig.constants import ACCURACY
+from ludwig.schema.config.model_metadata import ModelMetadata
+
```

Comment:
Will be adding more tests to this file as this process continues. For now, this test just makes sure the `ModelMetadata` object has all the intended attributes, none of the unintended attributes,  and constructs correctly.